resources/views/inputs/checkbox.blade.php

```diff
@@ -9,6 +9,7 @@
     'value'                 => null,
     'checked'               => false,
     'disabled'              => false,
+    'alpine'                => false,
```

Comment:
@Highjhacker Is there a reason why you're not using `<x-ark-checkbox>` component?
You can add the optional `alpine` argument on your [PR#533](https://github.com/ArkEcosystem/laravel-ui/pull/533) too.